lightcurve/alcdef.py

```diff
@@ -65,7 +76,14 @@ def add_data(self, jd, mag, magerr=None, airmass=None):
             data += " | %s" % (airmass)
         self.datas.append(data)
 
-    def get_response(self):
+    def tostring(self):
+        # error = []
+        # for k in REQUIRED_KEYWORDS:
+        #     if k not in self.metadata:
+        #         error.append(k)
+        # if len(error) > 0:
+        #   return error
```

Comment:
You could check this with set notation.  I think it's sufficient to just throw an error in this case that callers can catch.

```
missing_required = REQUIRED_KEYWORDS.difference(set(self.metadata.keys()))
if len(missing_required) > 0:
    raise RuntimeError('Missing required keywords: %s' % (', '.join(missing_required)))
```

Reply:
Done. Got the alert popup set up mentioned in previous email if I change the raise RuntimeError to return missing_required. Please let me know if you prefer just raise error.